StructureDefinition/England-Immunization.xml

```diff
@@ -0,0 +1,347 @@
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="0" />
+    </element>
+    <element id="Immunization.extension:parentPresent">
+      <path value="Immunization.extension" />
+      <sliceName value="parentPresent" />
+      <min value="0" />
+      <mustSupport value="false" />
+    </element>
+    <element id="Immunization.extension:vaccinationProcedure">
+      <path value="Immunization.extension" />
+      <sliceName value="vaccinationProcedure" />
+      <definition value="Where status=`completed` this is Mandatory&#xD;&#xA;This relates to the vaccine that was administered (procedure) and is a SNOMED CT from {{link:https://fhir.hl7.org.uk/ValueSet/UKCore-VaccinationProcedure}}&#xD;&#xA;&#xD;&#xA;In addition to the codes in the following codes are recommended.&#xD;&#xA;&#xD;&#xA;&lt;table&gt;&#xD;&#xA;&lt;thead&gt;&#xD;&#xA;&lt;th data-no-sort &gt;&#xD;&#xA;Vaccination Procedure&#xD;&#xA;&lt;/th&gt;&#xD;&#xA;&lt;th data-no-sort &gt;&#xD;&#xA;UK SNOMED self&#xD;&#xA;&lt;/th&gt;&#xD;&#xA;&lt;/thead&gt;&#xD;&#xA;&lt;tr&gt;&#xD;&#xA;&lt;td&gt;&#xD;&#xA;Administration of first dose of SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) vaccine&#xD;&#xA;&lt;/td&gt;&#xD;&#xA;&lt;td&gt;&#xD;&#xA;&lt;a href=&quot;https://termbrowser.nhs.uk/?perspective=full&amp;conceptId1=1324681000000101&quot; target=&quot;_blank&quot;&gt;1324681000000101&lt;/a&gt;&#xD;&#xA;&lt;/td&gt;&#xD;&#xA;&lt;/tr&gt;&#xD;&#xA;&lt;tr&gt;&#xD;&#xA;&lt;td&gt;&#xD;&#xA;Administration of second dose of SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) vaccine&#xD;&#xA;&lt;/td&gt;&#xD;&#xA;&lt;td&gt;&#xD;&#xA;&lt;a href=&quot;https://termbrowser.nhs.uk/?perspective=full&amp;conceptId1=1324691000000104&quot; target=&quot;_blank&quot;&gt;1324691000000104&lt;/a&gt;&#xD;&#xA;&lt;/td&gt;&#xD;&#xA;&lt;/tr&gt;&#xD;&#xA;&lt;/table&gt;&#xD;&#xA;&#xD;&#xA;&lt;br&gt;&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&quot;extension&quot;:  [&#xD;&#xA;        {&#xD;&#xA;            &quot;url&quot;: &quot;https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-VaccinationProcedure&quot;,&#xD;&#xA;            &quot;valueCodeableConcept&quot;: {&#xD;&#xA;                &quot;coding&quot;:  [&#xD;&#xA;                    {&#xD;&#xA;                        &quot;system&quot;: &quot;http://snomed.info/sct&quot;,&#xD;&#xA;                        &quot;code&quot;: &quot;1324681000000101&quot;,&#xD;&#xA;                        &quot;display&quot;: &quot;Administration of first dose of severe acute respiratory syndrome coronavirus 2 vaccine (procedure)&quot;&#xD;&#xA;                    }&#xD;&#xA;                ]&#xD;&#xA;            }&#xD;&#xA;        }&#xD;&#xA;    ],&#xD;&#xA;```" />
+      <min value="0" />
+      <mustSupport value="true" />
+      <binding>
+        <strength value="required" />
```

Comment:
Is there a reason that this is required? UK Core standard is for SNOMED based valuesets to be preferred

Reply:
Now changed to Preferred.